ConvivaExampleApp/src/androidTest/java/com/bitmovin/analytics/convivaanalyticsexample/ExampleInstrumentedTest.java

```diff
@@ -18,9 +18,5 @@
 public class ExampleInstrumentedTest {
     @Test
     public void useAppContext() {
-        // Context of the app under test.
-        Context appContext = InstrumentationRegistry.getTargetContext();
-
-        assertEquals("com.bitmovin.analytics.convivaanalyticsexample", appContext.getPackageName());
```

Comment:
Why not removing the whole test file? If you're going to add instrumentation tests, you will anyways create specific test files most likely.

Reply:
Good idea. Will remove it.

Reply:
removed the file